components/profile/LesswrongLink.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+import React, { useState } from "react";
+
+const LesswrongLink = () => {
```

Comment:
I think `LesswrongConnect` is a little better, isn't it? cuz `Link` has other meanings, whereas `Connect` sounds closer to auth. you could also call the component `LesswrongAuth`?